src/Compilers/VisualBasic/Portable/Binding/Binder_Symbols.vb

```diff
@@ -748,9 +804,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Private Shared Function ErrorTypeFromLookupResult(name As String, result As LookupResult, binder As Binder) As ErrorTypeSymbol
                 If result.Kind = LookupResultKind.Ambiguous AndAlso result.HasSingleSymbol AndAlso TypeOf result.Diagnostic Is AmbiguousSymbolDiagnostic Then
                     ' Special case: set of ambiguous symbols is stored in the diagnostics.
-                    Return Binder.GetErrorSymbol(name, result.Diagnostic, DirectCast(result.Diagnostic, AmbiguousSymbolDiagnostic).AmbiguousSymbols, result.Kind)
+                    Return binder.GetErrorSymbol(name, result.Diagnostic, DirectCast(result.Diagnostic, AmbiguousSymbolDiagnostic).AmbiguousSymbols, result.Kind)
```

Comment:
it is correctly matching the case of 'binder' parameter

---
In reply to: [71443670](https://github.com/dotnet/roslyn/pull/12596#discussion_r71443670) [](ancestors = 71443670)